packages/backend-defaults/package.json

```diff
@@ -82,38 +82,60 @@
     "test": "backstage-cli package test"
   },
   "dependencies": {
+    "@aws-sdk/abort-controller": "^3.347.0",
+    "@aws-sdk/client-codecommit": "^3.350.0",
+    "@aws-sdk/client-s3": "^3.350.0",
+    "@aws-sdk/credential-providers": "^3.350.0",
+    "@aws-sdk/types": "^3.347.0",
```

Comment:
Well now I really wanna be able to ship this as a service module :p

Reply:
Aye.